homeassistant/components/blink/alarm_control_panel.py

```diff
@@ -45,7 +45,6 @@ class BlinkSyncModuleHA(AlarmControlPanelEntity):
 
     _attr_icon = ICON
     _attr_supported_features = AlarmControlPanelEntityFeature.ARM_AWAY
-    _attr_name = None
```

Comment:
Yes - that's what I think we wanted.
There is a camera, alarm, binary sensors and sensors.  Without the "alarm" in this case and "camera" in the camera case, and names on the sensors every entity had the same name: DOMAIN Name and the entities were domain_name_2, domain_name_3,etc.
Right?

Reply:
depends on the value of name, if the value of name is the same, then yes, it would produce _2 _3. But then changing it all to a static name isn't the solution imo